auacm/app/static/html/scoreboard.html

```diff
@@ -1,7 +1,7 @@
 <div>
   <div class="row">
     <h1 class="pull-left">{{ competition.name }}</h1>
-    <div ng-if="isAdmin && active" class="dropdown pull-left margin-top margin-left">
+    <div ng-if="isAdmin && (active || !over)" class="dropdown pull-left margin-top margin-left">
```

Comment:
is this supposed to be `!ended`? What is the difference between `over` and `ended`?